sound/soc/codecs/rt711.c

```diff
@@ -405,13 +401,11 @@ static int rt711_set_jack_detect(struct snd_soc_component *component,
 	struct snd_soc_jack *hs_jack, void *data)
 {
 	struct rt711_priv *rt711 = snd_soc_component_get_drvdata(component);
-	struct snd_soc_dapm_context *dapm =
-		snd_soc_component_get_dapm(rt711->component);
 
 	while (!rt711->hw_init)
 		return -EAGAIN;
```

Comment:
you don't need a while here, an 'if' would do.

Reply:
Ok, I will modify it.